apps/src/codebridge/Workspace/WorkspaceAndConsole.tsx

```diff
@@ -0,0 +1,86 @@
+import Console from '@codebridge/Console';
+import Workspace from '@codebridge/Workspace';
+import {debounce} from 'lodash';
+import React, {useEffect, useMemo} from 'react';
+
+import HeightResizer from '@cdo/apps/templates/instructions/HeightResizer';
+
+import moduleStyles from './workspace.module.scss';
+
+// The top Y coordinate of the panel.  Above them is just the common site
+// header and then a bit of empty space.
+const PANEL_TOP_COORDINATE = 60;
+
+// A component that combines the Workspace and Console component into a single component,
+// with a horizontal resizer between them.
+// This component is intended to get us through the Python Lab pilot, where we just need a resizable
+// console. We will likely want a different pattern long-term, as we will want other components to
+// be resizable, and don't want to have to make a bunch of different combination components to achieve that.
+// We may want to investigate more general solution for this, such as a panel manager component.
+// We also will want resizing to be accessible, and the HeightResizer component only works with mouse and touch
+// events.
+const WorkspaceAndConsole: React.FunctionComponent = () => {
+  const [consoleHeight, setConsoleHeight] = React.useState(350);
+  const [columnHeight, setColumnHeight] = React.useState(800);
+
+  useEffect(() => {
+    const handleColumnResize = () => {
+      setColumnHeight(window.innerHeight - PANEL_TOP_COORDINATE);
+    };
+    handleColumnResize();
+
+    window.addEventListener('resize', debounce(handleColumnResize, 10));
+  }, []);
+
+  useEffect(() => {
+    normalizeConsoleHeight(consoleHeight);
+  }, [consoleHeight, columnHeight]);
+
+  const handleResize = (desiredHeight: number) => {
+    // While the horizontal resizer thinks it's resizing the content above it, which
+    // is the editor panel, we are actually storing the size of the console below it.
+    // That way, if the window resizes, the console stays the same height while the editor
+    // changes in height.
```

Comment:
Nice!

Reply:
I copied this from [JavalabPanels](https://github.com/code-dot-org/code-dot-org/blob/b128e412e16188bc063f61263ed8ac78331ca06c/apps/src/javalab/JavalabPanels.jsx#L108-L111), which means I think I copied it from you 😂